### PR TITLE
Safety moth expansion

### PR DESCRIPTION
**Pull request by JohnFulpWillard**
## About The Pull Request

2 new types of Safety Moth: Medical and Security.
Hardsuit helmet now protects against welding/flash.
They can now put their internals back on their hardsuit if they accidentally remove them.
Engineering safety moth now gets advanced insuls and a tablet.
Safety moth now gets a Captain encryption key, rather than just Engineering/Command

Also moves Safety moth stuff to their own file because it doesn't feel like it fits with all our other ERT stuff.
Also changes our nitrile insulated gloves to actually be nitrole gloves.

## Why It's Good For The Game

Don't forget the formaldehyde.

### CODE REVIEW

**Comment by JohnFulpWillard** [comment]
why are the checks failing !!

**Comment by Helianthus00** [comment]
@Horatio22 I'm curious what you think about these changes.

**Comment by Horatio22** [comment]
> 
> 
> @Horatio22 I'm curious what you think about these changes.

Pretty neutral, though I don't really understand the addition of a security safety moth since safety moth starts pacified. @AsphaltEvidence is the one to ask about how well it fits in with safety moth lore and whatnot

**Comment by JohnFulpWillard** [comment]
> 
> 
> > @Horatio22 I'm curious what you think about these changes.
> 
> Pretty neutral, though I don't really understand the addition of a security safety moth since safety moth starts pacified. @AsphaltEvidence is the one to ask about how well it fits in with safety moth lore and whatnot

Pacifists can still use disablers/batons, just nothing harmful, pacifism is a quirk in game and sec officers are fine with it. I added it so they can be like a lawyer-type ERT since we dont have one, telling sec what theyre doing wrong in brigging procedures, though maybe replacing with a centcom official would be better?
image

File: fulp_modules/features/ert/code/safety_moth.dm [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+	gloves = /obj/item/clothing/gloves/tackler/combat/insulated
+	shoes = /obj/item/clothing/shoes/combat/swat
+	belt = /obj/item/storage/belt/security/full
+	back = /obj/item/storage/backpack/ert/security
+	backpack_contents = list(
+		/obj/item/melee/baton/loaded = 1,
+		/obj/item/storage/box/handcuffs = 1,
+		/obj/item/storage/box/survival/engineer = 1,
+	)
+	l_pocket = /obj/item/tank/internals/oxygen
```
Comment (SgtHunk) [inline fulp_modules/features/ert/code/safety_moth.dm:118]: invalid left pocket

File: fulp_modules/features/ert/code/safety_moth.dm [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+	leader_role = /datum/antagonist/ert/safety_moth/security
+	roles = list(/datum/antagonist/ert/safety_moth/security)
+	polldesc = "an experienced Security Safety team"
+
+/// Antagonist status
+/datum/antagonist/ert/safety_moth
+	name = "Engineering Safety Moth"
+	role = "Engineering Safety Moth"
+	outfit = /datum/outfit/centcom/ert/safety_moth
+	var/specialty = SAFETY_MOTH_ENGINEER
```
Comment (SgtHunk) [inline fulp_modules/features/ert/code/safety_moth.dm:29]: what's the point of this? the defines show that it's the same thing as the name/role of the datum. Also christ the spaces

File: fulp_modules/features/ert/code/safety_moth.dm [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+	. = ..()
+	if(visualsOnly)
+		return
+	var/obj/item/radio/R = H.ears
+	R.keyslot = new /obj/item/encryptionkey/heads/captain
+	R.recalculateChannels()
+
+//Eng
+/obj/item/clothing/suit/space/safety_moth
+	name = "Safety Moth safety vest"
```
Comment (SgtHunk) [inline fulp_modules/features/ert/code/safety_moth.dm:138]: bobux no lowercase

File: fulp_modules/features/ert/code/safety_moth.dm [hunk 4]
```diff
@@ -0,0 +1,185 @@
...
+	inhand_icon_state = "hazard"
+	flags_inv = null
+	armor = list(MELEE = 50, BULLET = 40, LASER = 40, ENERGY = 50, BOMB = 50, BIO = 100, RAD = 100, FIRE = 100, ACID = 80)
+	allowed = list(/obj/item/gun, /obj/item/ammo_box, /obj/item/ammo_casing, /obj/item/melee/baton, /obj/item/restraints/handcuffs, /obj/item/tank/internals, /obj/item/toy/plush/moth)
+	resistance_flags = FIRE_PROOF
+	heat_protection = CHEST|GROIN|LEGS|FEET|ARMS|HANDS
+	max_heat_protection_temperature = FIRE_IMMUNITY_MAX_TEMP_PROTECT
+	slowdown = 0
+/obj/item/clothing/head/helmet/space/safety_moth
+	name = "Safety Moth safety hardhat"
```
Comment (SgtHunk) [inline fulp_modules/features/ert/code/safety_moth.dm:150]: lowercase

File: fulp_modules/features/ert/code/safety_moth.dm [hunk 5]
```diff
@@ -0,0 +1,185 @@
...
+	armor = list(MELEE = 50, BULLET = 40, LASER = 40, ENERGY = 50, BOMB = 50, BIO = 100, RAD = 100, FIRE = 100, ACID = 80)
+	heat_protection = HEAD
+	max_heat_protection_temperature = FIRE_IMMUNITY_MAX_TEMP_PROTECT
+/obj/item/clothing/shoes/magboots/safety_moth
+	desc = "A special pair of Magnetic boots which doesn't weight you down." // Do they even need this? They're a moth ffs.
+	name = "safety magboots"
+	slowdown_active = 1
+//Med
+/obj/item/clothing/suit/space/safety_moth/med
+	name = "Safety Moth medical jacket"
```
Comment (SgtHunk) [inline fulp_modules/features/ert/code/safety_moth.dm:163]: lowercase

File: fulp_modules/features/ert/code/safety_moth.dm [hunk 6]
```diff
@@ -0,0 +1,185 @@
...
+	desc = "The standard issue safety jacket belonging to Safety Moth themselves. Functions like a hardsuit and offers superb protection against environmental hazards."
```
Comment (SgtHunk) [inline fulp_modules/features/ert/code/safety_moth.dm:164]: weird wording

File: fulp_modules/features/ert/code/safety_moth.dm [hunk 7]
```diff
@@ -0,0 +1,185 @@
...
+	icon_state = "labcoat_paramedic"
+	inhand_icon_state = "labcoat_paramedic"
+/obj/item/clothing/head/helmet/space/safety_moth/med
+	name = "Safety Moth medical cap"
+	desc = "The standard issue safety cap belonging to Safety Moth themselves. Functions like a hardsuit helmet and offers superb protection against environmental hazards."
```
Comment (SgtHunk) [inline fulp_modules/features/ert/code/safety_moth.dm:169]: read above

File: fulp_modules/features/ert/code/safety_moth.dm [hunk 8]
```diff
@@ -0,0 +1,185 @@
...
+	icon_state = "paramedicsoft"
+//Sec
+/obj/item/clothing/suit/space/safety_moth/sec
+	name = "Safety Moth riot suit"
+	desc = "The standard issue riot suit belonging to Safety Moth themselves. Functions like a hardsuit, resists shoves, and offers superb protection against environmental hazards."
```
Comment (SgtHunk) [inline fulp_modules/features/ert/code/safety_moth.dm:174]: read above

File: fulp_modules/features/ert/code/safety_moth.dm [hunk 9]
```diff
@@ -0,0 +1,185 @@
...
+	icon_state = "riot"
+	inhand_icon_state = "riot"
+	clothing_flags = BLOCKS_SHOVE_KNOCKDOWN
```
Comment (SgtHunk) [inline fulp_modules/features/ert/code/safety_moth.dm:177]: this overwrites the space suit's clothing flags, resulting in a suit without pressure protection or without being thick.
Yes I made the same error with the sec commander ert hardsuit

File: fulp_modules/features/ert/code/safety_moth.dm [hunk 10]
```diff
@@ -0,0 +1,185 @@
...
+/obj/item/clothing/head/helmet/space/safety_moth/sec
+	name = "Safety Moth riot helmet"
+	desc = "The standard issue riot helmet belonging to Safety Moth themselves. Functions like a hardsuit helmet and offers superb protection against environmental hazards."
```
Comment (SgtHunk) [inline fulp_modules/features/ert/code/safety_moth.dm:180]: lowercase and wording

File: fulp_modules/features/ert/code/safety_moth.dm [hunk 11]
```diff
@@ -0,0 +1,185 @@
...
+	desc = "The standard issue safety hardhat belonging to Safety Moth themselves. Functions like a hardsuit helmet and offers superb protection against environmental hazards."
+	icon_state = "hardhat0_yellow"
+	resistance_flags = FIRE_PROOF
+	armor = list(MELEE = 50, BULLET = 40, LASER = 40, ENERGY = 50, BOMB = 50, BIO = 100, RAD = 100, FIRE = 100, ACID = 80)
+	heat_protection = HEAD
+	max_heat_protection_temperature = FIRE_IMMUNITY_MAX_TEMP_PROTECT
+/obj/item/clothing/shoes/magboots/safety_moth
+	desc = "A special pair of Magnetic boots which doesn't weight you down." // Do they even need this? They're a moth ffs.
+	name = "safety magboots"
+	slowdown_active = 1
```
Comment (SgtHunk) [inline fulp_modules/features/ert/code/safety_moth.dm:160]: > which doesn't weight you down
> slowdown when active: 1 (same as a space suit)

lol

File: fulp_modules/features/ert/code/safety_moth.dm [hunk 12]
```diff
@@ -0,0 +1,185 @@
...
+	if(visualsOnly)
+		return
+	var/obj/item/radio/R = H.ears
+	R.keyslot = new /obj/item/encryptionkey/heads/captain
+	R.recalculateChannels()
+
+//Eng
+/obj/item/clothing/suit/space/safety_moth
+	name = "Safety Moth safety vest"
+	desc = "The standard issue safety vest of the Safety Moth. Functions like a hardsuit and offers superb protection against environmental hazards."
```
Comment (SgtHunk) [inline fulp_modules/features/ert/code/safety_moth.dm:139]: wording

**Review by SgtHunk — CHANGES_REQUESTED** [review]

File: fulp_modules/features/ert/code/safety_moth.dm (hunk 2 above)
Comment (JohnFulpWillard) [reply fulp_modules/features/ert/code/safety_moth.dm:29]: I was originally gonna use it for their equipment like med ert and headset

File: fulp_modules/features/ert/code/safety_moth.dm (hunk 3 above)
Comment (JohnFulpWillard) [reply fulp_modules/features/ert/code/safety_moth.dm:138]: Uppercase for names :/

**Pull request closed without merging**